audiomentations/augmentations/air_absorption.py

```diff
@@ -0,0 +1,172 @@
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+import numpy as np
+import librosa
+
+
+def next_power_of_2(x: int) -> int:
+    """
+    taken jhoyla's answer here:
+    https://stackoverflow.com/questions/14267555/find-the-smallest-power-of-2-greater-than-or-equal-to-n-in-python
+    """
+    return 1 if x == 0 else 2 ** (x - 1).bit_length()
+
+
+class AirAbsorption(BaseWaveformTransform):
+    """
+    Applies a wind absorption transform parametrized by temperature, humidity, and distance
+    of source.
+
+    This is not a scientifically accurate transform but basically applies a uniform
+    filterbank with attenuations given by:
+
+    att = exp(- distance * absorption_coefficient)
+
+    where distance is the microphone-source assumed distance in meters and `absorption_coefficient`
+    is adapted from a lookup table by pyroomacoustics [1]. It can also be seen as a lowpass filter
+    with variable octave attenuation.
+
+    Note: This only "simulates" the dampening of high frequencies, and does not  attenuate according to the distance law. Gain augmentation needs
+    to be done separately.
+
+    [1] https://github.com/LCAV/pyroomacoustics
+    """
+
+    supports_multichannel = True
+
+    # Table of air absorption coefficients adapted from `pyroomacoustics`.
+    # The last two columns have been extrapolated with `curve_fit`
+    air_absorption_table = {
+        "10C_30-50%": [
```

Comment:
Maybe specify what the different parts of the name "10C_30-50%" mean ? I guess that it is temperature, distance and humidity percentage, bit I think it would be more clear if you specify that as a comment.

Reply:
This is something the user shouldn't see. I specify what the values are in the docstring.

Reply:
I still think it would be more clear if you would say something like:
# Table of air absorption coefficients adapted from `pyroomacoustics`.
# The keys describe the temperature and the humidity range.
# The last two columns have been extrapolated with `curve_fit`

At first sight, I didn't understand that 30-50% was a range, I thought it was two different values.

Reply:
I see what you mean. Will fix it.